frontend/src/features/workspace/components/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -67,7 +71,11 @@ export const WorkspaceHeader = ({
         >
           Create form
         </Button>
-        <Button isFullWidth={isMobile} onClick={handleOpenWhatsNewDrawer}>
+        <Button
+          isFullWidth={isMobile}
+          onClick={handleOpenWhatsNewDrawer}
+          variant={whatsNewButtonVariant}
+        >
```

Comment:
question: this is temporary right (until workspace header is done)? any reason why we need the `variant` prop?

Reply:
Yes this is temporary because the admin header is not up yet for me to add the notification icon. Hence I used the button variant to represent whether or not there is a new feature update that the form admin user has not seen yet.

Reply:
gotcha. maybe need a TODO to signify that it is temporary